model/src/guild/role.rs

```diff
@@ -17,10 +18,29 @@ pub struct Role {
     pub tags: Option<RoleTags>,
 }
 
+impl Ord for Role {
+    /// Roles are ordered primarily by position. Discord does not provide guarentees that role
```

Comment:
This currently reads like a code comment. If this should be visible documentation then this should be written in a more technical fashion with an example of how to use it. Let's just go that route, actually.